pandas/plotting/_core.py

```diff
@@ -2818,18 +2818,42 @@ def kde(self, bw_method=None, ind=None, **kwds):
 
     def area(self, x=None, y=None, **kwds):
         """
-        Area plot
+        Draw an area plot of the input series using matplotlib.
+
+        An area plot displays graphically quantitative data.
+        This function wraps the matplotlib area function.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
-        `**kwds` : optional
+        x : label or position, optional
+            Coordinates for X point.
+        y : label or position, optional
+            Coordinates for Y point.
+        kwds : optional
```

Comment:
As mentioned in other PRs, you should use `**kwds` instead. Sorry for the great big confusion here.

Reply:
Using `**kwds` raises errors in the current version of the validator:

	Errors in parameters section
		Parameters {'kwds'} not documented
		Unknown parameters {'**kwds'}

I think it's important to be consistent between the validator and the code.

Of course, once the validator is updated, I agree this must be changed.

Reply:
You can ignore that error in the validation script for now.

Reply:
Ok, updated!